roles/power_aix_bootstrap/tasks/dnf_install.yml

```diff
@@ -42,143 +42,91 @@
 # { Excute this block if dnf is not installed and the AIX version is equal or greater than 7.3
 - block:
 
-     # DOWNLOAD (local) install images
-     - name: Check for rpm_src file on localhost
-       stat:
-          path: "{{ download_dir }}/{{ rpm_src }}"
-       register: rpm_dest
-       delegate_to: localhost
+     - name: Set the python interpreter to python3
+       set_fact:
+          ansible_python_interpreter: /usr/bin/python3
 
-     - name: Download latest rpm (installp) fileset to the controller.
-       get_url:
-          url: "http://public.dhe.ibm.com/aix/freeSoftware/aixtoolbox/INSTALLP/ppc/{{ rpm_src }}"
-          dest: "{{ download_dir }}/{{ rpm_src }}"
-          mode: '0644'
-       delegate_to: localhost
-
-#     - name: Check for "{{ dnf_src_71_72 }}" file on localhost
-#       stat:
-#          path: "{{ download_dir }}/{{ dnf_src_71_72 }}"
-#       register: dnf_71_72_dest
-#       delegate_to: localhost
-#       when: not aix73
-
-     - name: Check for "{{ dnf_src_73 }}" file on localhost
-       stat:
-          path: "{{ download_dir }}/{{ dnf_src_73 }}"
-       register: dnf_73_dest
-       delegate_to: localhost
-
-#     - name: Download dnf bundle file for 7.1 and 7.2 to the controller
-#       get_url:
-#          url: "http://public.dhe.ibm.com/aix/freeSoftware/aixtoolbox/ezinstall/ppc/{{ dnf_src_71_72 }}"
-#          dest: "{{ download_dir }}/{{ dnf_src_71_72 }}"
-#          mode: '0644'
-#       when: not aix73 and not dnf_71_72_dest.stat.exists
-#       delegate_to: localhost
-
-     - name: Download dnf bundle file "{{ dnf_src_73 }}" for 7.3 to the controller
+     - name: Download latest dnf bundle file "{{ dnf_src_73 }}" for 7.3 to the controller
```

Comment:
so this is specific for AIX7.3... what would happen for AIX7.2 user? do we need to add some validation to block?